docsource/modules150-160.rst

```diff
@@ -70,7 +70,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | |del| base_address_city                         |                      |                                                 |
```

Comment:
```suggestion
| |del| base_address_city                         | Done                 | Merged into base_address_extended            |
```

As this has been already tackled in https://github.com/OCA/OpenUpgrade/blob/16.0/openupgrade_scripts/apriori.py#L35